administrator/language/en-GB/en-GB.com_installer.ini

```
@@ -123,6 +123,7 @@ COM_INSTALLER_MSG_DISCOVER_PURGEDDISCOVEREDEXTENSIONS="Cleared discovered extens
 COM_INSTALLER_MSG_ERROR_CANT_CONNECT_TO_UPDATESERVER="Can't connect to %s"
 COM_INSTALLER_MSG_INSTALL_ENTER_A_URL="Please enter a URL"
 COM_INSTALLER_MSG_INSTALL_INVALID_URL="Invalid URL"
+COM_INSTALLER_MSG_INSTALL_INVALID_URL_SCHEMA="Please enter a valid URL starting with http or https"
```

```suggestion
COM_INSTALLER_MSG_INSTALL_INVALID_URL_SCHEMA="Please enter a valid URL starting with http or https."
```
Its a sentence so it needs to end with a .